apps/api/src/services/data-provider/yahoo-finance/yahoo-finance.service.ts

```diff
@@ -197,17 +197,17 @@ export class YahooFinanceService implements DataProviderInterface {
           // filter out undefined symbols
           return quote.symbol;
         })
-        .filter(({ quoteType }) => {
+        .filter(({ quoteType, symbol }) => {
           return (
-            quoteType === 'CRYPTOCURRENCY' ||
+            (quoteType === 'CRYPTOCURRENCY' && this.isCryptoCurrency(symbol)) ||
             quoteType === 'EQUITY' ||
             quoteType === 'ETF'
           );
         })
         .filter(({ quoteType, symbol }) => {
           if (quoteType === 'CRYPTOCURRENCY') {
-            // Only allow cryptocurrencies in USD
-            return symbol.includes('USD');
+            // Only allow cryptocurrencies in USD and EUR
```

Comment:
I only want to support _Bitcoin_ once (in `USD`). Next users would add `CAD`, then `CHF` and so on... A lot of redundant data.

Reply:
Ok, makes sense.